doc/nrf/links.txt

```diff
@@ -1348,3 +1354,13 @@
 .. _`Setting up the SDK for Amazon Sidewalk`: https://nrfconnect.github.io/sdk-sidewalk/setting_up_sidewalk_environment/setting_up_sdk.html
 
 .. _`Community Best Practice for Copyright Notice`: https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/
+
+.. _`Mobile IoT Deployment Map`: https://www.gsma.com/iot/deployment-map/
+
+.. _`Onomondo SoftSIM intergration with the nRF91 Series`: https://help.onomondo.com/en/how-to-integrate-the-onomondo-softsim-with-the-nrf9160-from-nordic-semiconductor
+
+.. _`Certifying a Cellular IoT Device`: https://blog.nordicsemi.com/getconnected/certifying-a-cellular-iot-device/
+
+.. _`Accelerate Cellular Product Development`: https://youtu.be/b1YW1J4aUps
+
+.. _`Cellular IoT asset tracking`: https://youtu.be/V-WlGbyRDZg
```

Comment:
There is one other youtube.com link in links.txt, so these three links can be grouped together.